jcli/src/jcli_app/transaction/mod.rs

```diff
@@ -158,11 +164,22 @@ impl Transaction {
 }
 
 fn display_id(common: common::CommonTransaction) -> Result<(), Error> {
+    eprintln!("DEPRECATED: use 'data-for-witness' instead\n");
```

Comment:
good idea printing on stderr, the message won't be caught by scripts.

however, no need for the extra `\n`.

```suggestion
    eprintln!("DEPRECATED: use 'data-for-witness' instead");
```

Reply:
`stderr` was suggested by @vincenthz https://github.com/input-output-hk/jormungandr/pull/1069#issuecomment-549266328 :)

newline removed.